cibuildwheel/__main__.py

```diff
@@ -65,27 +66,33 @@ def main() -> None:
 
     parser.add_argument(
         "--output-dir",
+        type=Path,
         help="Destination folder for the wheels. Default: wheelhouse.",
     )
 
     parser.add_argument(
         "--config-file",
         default="",
         help="""
-            TOML config file. Default: "", meaning {package}/pyproject.toml,
-            if it exists.
+            TOML config file. Default: "", meaning {package}/pyproject.toml, if
+            it exists. To refer to a project inside your project, use {package};
+            this matters if you build from an SDist.
         """,
     )
 
     parser.add_argument(
         "package_dir",
```

Comment:
Could this argument be renamed to `package`? since it's not a dir when it's an sdist archive.

Reply:
I renamed it in the CLI, left it in the code because it quickly is resolved to a dir in both cases.